h/app.py

```diff
@@ -42,6 +22,28 @@ def create_app(global_config, **settings):
 
     config.add_subscriber('h.subscribers.add_renderer_globals',
                           'pyramid.events.BeforeRender')
+    config.add_subscriber('h.subscribers.set_user_from_oauth',
+                          'pyramid.events.NewRequest')
+
+    config.add_tween('h.tweens.csrf_tween_factory')
+    config.add_tween('h.tweens.auth_token')
+
+    config.include(__name__)
+
+    app = config.make_wsgi_app()
+    app = permit_cors(app,
```

Comment:
Let's not enable CORS for the entire application. That allows anyone to load any of our asset javascript cross-origin, which rather defeats the point of CORS in the first place. It shouldn't be too hard to add a PATH_INFO restriction to the `permit_cors` middleware.